src/test_l1_dsHdmiIn.c

```diff
@@ -2306,7 +2312,7 @@ void test_l1_dsHdmiIn_positive_dsGetEdid2AllmSupport(void) {
  * @note Testing for the `dsERR_OPERATION_NOT_SUPPORTED` and `dsERR_OPERATION_FAILED` might be challenging since it requires a specific scenario where the attempted operation is not supported or fails due to other reasons.
  * 
  */
-void test_l1_dsHdmiIn_negative_dsGetEdid2AllmSupport(void) {
+void test_l1_dsHdmiIn_negative_dsGetEdid2AllmSupport_sink(void) {
     gTestID = 50;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     bool allmSupport;
```

Comment:
test_l1_dsHdmiIn_register - In the register function, the changes to the function names are not being reflected. Additionally, please check the compilation part as well.

Reply:
Done